prisma/migrations/20230609100223_swish/migration.sql

```diff
@@ -0,0 +1,20 @@
+-- CreateTable
+CREATE TABLE `SwishOrder` (
+    `ID` INTEGER NOT NULL AUTO_INCREMENT,
+    `KID` VARCHAR(15) NOT NULL,
+    `instructionUUID` VARCHAR(36) NOT NULL,
+    `donorID` INTEGER NOT NULL,
+    `donationID` INTEGER NULL,
+    `status` ENUM('PAID', 'DECLINED', 'ERROR', 'CANCELLED') NULL,
```

Comment:
Mhm, TIL you cannot have standalone enums in mysql, they always belong to a specific table